src/features/rewards/disabledPanel/index.tsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledTitle,
+  StyledOff,
+  StyledText,
+  StyledLink,
+  StyledTitleWrapper,
+  StyledTextWrapper
+} from './style'
+
+import { getLocale } from '../../../helpers'
+
+export interface Props {
+  id?: string
+  isPrivate?: boolean
+  onLinkOpen: () => void
+}
+
+export default class DisabledPanel extends React.PureComponent<Props, {}> {
+  render () {
+    const { id, isPrivate, onLinkOpen } = this.props
+
+    return (
+      <StyledWrapper id={id}>
+        <StyledTitleWrapper>
+          <StyledTitle>
+            {getLocale('disabledPanelTitle')}
+          </StyledTitle>
+          <StyledOff>
+            {getLocale('disabledPanelOff')}
+          </StyledOff>
+        </StyledTitleWrapper>
+        <StyledTextWrapper>
```

Comment:
Do we need wrapper and then text? Can we just have text?

Reply:
I have a few rules on on `StyledTextWrapper` that are being used. 

In the case where I have text containing two components, in this cased `StyledText` and `StyledLink`, I prefer to have a parent container

Reply:
but isn't `StyledLink` inside `StyledText`

Reply:
so `StyledTextWrapper` has only one direct child component

Reply:
Lol yes it is, I misread my own code as it's a pattern I frequently use, I can update